package.json

```diff
@@ -1420,14 +1420,16 @@
             "browser",
             "tab",
             "singleton",
-            "external"
+            "external",
+            "customEditor"
           ],
           "markdownDescription": "The default PDF viewer.",
           "enumDescriptions": [
             "Open PDF with the default web browser.",
             "Open PDF with the built-in tab viewer.",
             "Open PDF with the built-in tab viewer, reveal existing one if possible.",
-            "[Experimental] Open PDF with the external viewer set in \"View > Pdf > External: command\"."
+            "[Experimental] Open PDF with the external viewer set in \"View > Pdf > External: command\".",
+            "Open PDF with the built-in tab viewer, treating it as a custom editor."
```

Comment:
General users may not know what is a custom editor. I suggest rephrase it to something like "use vscode custom editor api" or so.

Reply:
Also, I'd suggest to add an "experimental" flag.

Reply:
I added the flag and modified the description. Feel free to change the description again, if you have something specific in mind.